### ISSUE TITLE
Trojan:Script/Wacatac.B!m!

### ISSUE THREAD

**Issue by eapo** [AdamISZ/aut-ct#12]
<img width="408" alt="image" src="https://github.com/AdamISZ/aut-ct/assets/4000929/8609d2fd-9205-4e87-9c6e-28089632640a">

**Comment by AdamISZ** [comment]
Yes, I saw that myself. It is a false positive but it was stupid of me really to just leave it there because *I* happen to know it's safe ... I'll remove it from the release now, and at some later point figure out how to get rid of that, if it's easy.

**Comment by AdamISZ** [comment]
There are many similar forum threads to these three. The pattern is very clear: this is a machine learning triggered generic warning, classified as "Severe", which is triggered by a large range of stuff that you can't really predict (because it's machine learning that's used to classify!):

https://www.reddit.com/r/antivirus/comments/121947n/windows_defender_claims_an_old_zip_file_has/

https://www.reddit.com/r/rust/comments/14ai6pf/comment/jobrg1s/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

https://github.com/rust-lang/rust/issues/88297

Here's the typical user experience; massive amounts of time wasted on this "defence":

https://forums.chaos.com/forum/v-ray-for-3ds-max-forums/v-ray-for-3ds-max-problems/1181782-trojan-script-wacatac-h-ml

Unimaginably simple 2kB executable detected as the same "Wacatac" trojan:

https://board.flatassembler.net/topic.php?t=21786

**Comment by AdamISZ** [comment]
Also quite notable that the new `auditverifier` executable I added does not trigger the trojan warning, though it uses the same codebase.

Through a lot of internet searching I am not finding any concrete, good answer as to how to remove this warning for the large majority of Windows users that are using its default antivirus; I'm just finding an enormous amount of other people with the same complaint. It's notable that these complaints have been around for years now, so it's unlikely that they will suddenly stop. It is also notable though, that many people report that if they run a *different* virus scanning tool (not Defender), they find nothing.

**Comment by AdamISZ** [comment]
After reaching out to some other developers, it appears that the only solution to this type of problem is getting a signing certificate/code signing. However I will not do that (for both ethical and practical reasons), so I'll close this issue on the basis that distributing binaries for Windows is not possible.

(It may also end up being infeasible to distribute binaries for MacOS too, but I'll wait until I get more info on that).
